src/modules/form/components/DynamicFormLayout.tsx

```diff
@@ -1,18 +1,20 @@
-import { Grid } from '@mui/material';
+import { Grid, GridProps } from '@mui/material';
 import { ReactNode } from 'react';
 
 export interface DynamicFormLayoutProps {
   errors?: ReactNode;
   children: ReactNode;
   saveButtons?: ReactNode;
   variant?: 'standard' | 'without_top_level_cards';
+  GridProps?: GridProps;
 }
 
 const DynamicFormLayout = ({
   errors,
   children,
   saveButtons,
   variant = 'standard',
+  GridProps,
```

Comment:
Support passing `GridProps` because it's currently used on the `DynamicView` in two places, both legacy-referrals-world:
- https://github.com/greenriver/hmis-frontend/blob/46530d98b3ab4a1091bec0a79d575c3d30f9aa33/src/modules/legacyReferrals/components/admin/AdminReferralPostingForm.tsx#L81-L85
- https://github.com/greenriver/hmis-frontend/blob/46530d98b3ab4a1091bec0a79d575c3d30f9aa33/src/modules/legacyReferrals/components/ProjectReferralPostingForm.tsx#L88-L95

we could also support this as another variant? Or, we could just remove support for this. If I'm understanding correctly, while we still support some views on legacy referrals, neither of these forms would be shown to users anymore (once #8631 is in place, soon)